liberty-maven-plugin/src/it/outputdirectory-no-parent-pom-it/pom.xml

```diff
@@ -0,0 +1,174 @@
+        <dependency>
+            <groupId>commons-httpclient</groupId>
+            <artifactId>commons-httpclient</artifactId>
+            <version>3.1</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.9</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.openliberty.tools.it</groupId>
+            <artifactId>loose-config-fragment-it</artifactId>
+            <version>1.0-SNAPSHOT</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <version>3.0.0</version>
+                <configuration>
+                    <failOnMissingWebXml>false</failOnMissingWebXml>
+                    <packagingExcludes>pom.xml</packagingExcludes>
+                </configuration>
+            </plugin>
+			<plugin>
+				<groupId>io.openliberty.tools</groupId>
+                <artifactId>liberty-maven-plugin</artifactId>
+                <version>@pom.version@</version>
+                <configuration>
+                    <stripVersion>true</stripVersion>
+                    <assemblyArtifact>
+                        <groupId>${project.groupId}</groupId>
+                        <artifactId>assembly-server</artifactId>
+                        <version>${project.version}</version>
+                        <type>zip</type>
+                    </assemblyArtifact>
+                    <serverName>test</serverName>
+                    <configFile>src/main/liberty/config/server.xml</configFile>
+                    <appsDirectory>apps</appsDirectory>
+                    <installAppPackages>project</installAppPackages>
+                    <looseApplication>true</looseApplication>
+                    <stripVersion>true</stripVersion>
+                    <packageFile>${package.file}</packageFile>
+                    <include>${packaging.type}</include>
+                </configuration>
+                <executions>
+					<execution>
+						<id>stop-before-clean</id>
+						<phase>pre-clean</phase>
+						<goals>
+							<goal>stop</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>create-server</id>
+						<phase>prepare-package</phase>
+						<goals>
+							<goal>install-server</goal>
+							<goal>create-server</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>install-feature</id>
+						<phase>prepare-package</phase>
+						<goals>
+							<goal>install-feature</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>install-apps</id>
+						<phase>package</phase>
+						<goals>
+							<goal>install-apps</goal>
+						</goals>
+					</execution>
+					<execution>
+						<id>package-server</id>
```

Comment:
Run package during the integration-test phase when the server is running to check if the outputDir is working.